tests/kokkos-based/matrix_frob_norm_kokkos.cpp

```diff
@@ -87,7 +87,7 @@ TEST_F(blas2_signed_float_fixture, kokkos_matrix_frob_norm_trivial_empty)
 {
   std::vector<value_type> v;
 
-  constexpr auto de = std::experimental::dynamic_extent;
+  constexpr auto de = std::dynamic_extent;
```

Comment:
Does CI not actually test this test?  (Please see comment above about `std::dynamic_extent`.)  If so, then should we consider using the namespace macros instead of `std` explicitly?